net/apfree-wifidog/Makefile

```diff
@@ -60,6 +60,7 @@ define Package/apfree-wifidog/install
 	$(INSTALL_BIN) ./files/wifidogx.init $(1)/etc/init.d/wifidogx
 	$(INSTALL_DIR) $(1)/etc/config
 	$(CP) ./files/wifidogx.conf $(1)/etc/config/wifidogx
+	$(INSTALL_DIR) $(1)/etc/wifidogx
```

Comment:
Same here ;)

Reply:
@1715173329 

I apologize for any confusion caused by the new directory '/etc/wifidogx.' This directory is intended to store the local portal page for local authentication when it is enabled. However, maybe it is not the appropriate location for holding the local portal page.